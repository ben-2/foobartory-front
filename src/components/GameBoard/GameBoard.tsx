import styles from './GameBoard.module.css';
import { FormattedMessage } from 'react-intl';

export function GameBoard() {
  return (
    <>
      <div className={styles.subtitle}>
        GAME BOARD
      </div>
    </>
  );
}
