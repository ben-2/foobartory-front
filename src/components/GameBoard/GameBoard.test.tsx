import { GameBoard } from './GameBoard';
import { shallow } from 'enzyme';

describe('<GameBoard />', () => {
  it('renders a game board when the game starts', () => {
    const wrapper = shallow(<GameBoard />)
    expect(wrapper).toMatchSnapshot();
  })
});
