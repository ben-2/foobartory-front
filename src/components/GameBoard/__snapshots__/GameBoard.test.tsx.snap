// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GameBoard /> renders a game board when the game starts 1`] = `ShallowWrapper {}`;
